### ISSUE TITLE
Default ifort at NERSC does not support -openmp but default f2py (numpy v1.12.1) uses only it.

### ISSUE THREAD

**Issue by JulienPeloton** [JulienPeloton/s4cmb#12]
This issue has been brought by Maria S.
By default on Cori, an intel environment is loaded.
NERSC recently switched on using a new default version of ifort (the fortran intel compiler) which doesn’t support anymore the option -openmp (but -qopenmp).

Unfortunately, f2py (which is used for pyslalib or s4cmb for example) didn’t implement this change (it uses -openmp by default for Intel), hence there will be a problem at compilation.
You check that just typing the following in your terminal:

`$ f2py -c --help-fcompiler`

A quick workaround if you really want to use Intel is to unload the current module, and load a previous version:

`$ module unload intel`
`$ module load intel/17.0.2.174`

You see a deprecation warning, but that should work. 
Another option (which I do) is to use GNU compilers only.

I leave this issue open in case someone still has a problem.

**Comment by JulienPeloton** [comment]
Digging a bit more, I found that actually this problem has been solved at least in numpy 1.13.1. For numpy version >= 1.13.1, f2py uses now either -openmp or -qopenmp depending on the intel compiler. Unfortunately when you load the latest python module at NERSC (python/2.7-anaconda-4.4) the numpy version is 1.12.1!

To avoid any problem, numpy version >= 1.13.1 should be loaded as default at NERSC. I opened a ticket.
